apps/chat/src/components/Header/Logo.tsx

```diff
@@ -56,7 +60,7 @@ export const Logo = () => {
   return (
     <button
       onClick={handleLogoClick}
-      disabled={messageIsStreaming}
+      disabled={messageIsStreaming || isNewConversationDisabled}
```

Comment:
Need to check `isNewConversationDisabled` into `createNewConversation`. Not here
```
if (!areConversationsLoaded || isNewConversationDisabled) return;
```